lib/utils.js

```diff
@@ -459,6 +469,55 @@ function getTrackerEntryDomain (trackerEntries, domain, skipSubdomains = 0) {
     return null
 }
 
+/**
+ * Generate a mapping from tracker domain to a list of related aliases (via CNAME)
+ * @param {TDS} tds Tracker blocklist
+ * @returns {Map<string, string[]>} domain mapping
+ */
+function generateTrackerDomainAliasMapping (tds) {
```

Comment:
I renamed to `generateRequestDomainsByTrackerDomain`. I stuck with `generate` because I don't want it to sound like a simple getter. `generate` recognizes that more processing is being done.